sys/evtimer/evtimer.c

```diff
@@ -172,14 +204,13 @@ void evtimer_init(evtimer_t *evtimer, void(*handler)(void*))
     evtimer->events = NULL;
 }
 
-#if ENABLE_DEBUG == 1
```

Comment:
Due to this you will get link errors, if called externally, but `ENABLE_DEBUG` here is 0.